bin/inflo

```diff
@@ -55,6 +81,35 @@ def main():
     server_list.add_argument('-r', '--raw', action='store_true', help='Choose for raw json output.')
     server_list.set_defaults(func=invoke_server_list)
 
+    vm_info = subparsers.add_parser('vm-info', help='Get virtual machine info from given vm-id.')
+    vm_info.add_argument('-a', '--api-key', type=str, help='API key.', default=None)
+    vm_info.add_argument('-i', '--customer-id', type=str, help='Client ID.', default=None)
+    vm_info.add_argument('-r', '--raw', action='store_true', help='Choose for raw json output.')
+    vm_info.add_argument('-m', '--vm-id', type=str, help='Virtual machine ID.', default=None)
+    vm_info.set_defaults(func=invoke_get_vm_info)
+
+    vm_snapshots = subparsers.add_parser('vmsnapshot-list', help='Get virtual machine snapshot list.')
+    vm_snapshots.add_argument('-a', '--api-key', type=str, help='API key.', default=None)
+    vm_snapshots.add_argument('-i', '--customer-id', type=str, help='Client ID.', default=None)
+    vm_snapshots.add_argument('-m', '--vm-id', type=str, help='Virtual machine ID.', default=None)
+    vm_snapshots.set_defaults(func=invoke_get_vm_snapshots)
+
+    vm_backup = subparsers.add_parser('vmbackup-list', help='Get virtual machine backup list.')
+    vm_backup.add_argument('-a', '--api-key', type=str, help='API key.', default=None)
+    vm_backup.add_argument('-i', '--customer-id', type=str, help='Client ID.', default=None)
+    vm_backup.add_argument('-m', '--vm-id', type=str, help='Virtual machine ID.', default=None)
```

Comment:
I think we need to leave here only multisymbol key, cause '-m' is not clear for me as user and not memorable.